include/micm/process/process_set.hpp

```diff
@@ -315,15 +333,18 @@ namespace micm
       {
         for (std::size_t i_ind = 0; i_ind < number_of_reactants_[i_rxn]; ++i_ind)
         {
-          std::vector<double> d_rate_d_ind(L, 0);
-          for (std::size_t i_cell = 0; i_cell < L; ++i_cell)
-            d_rate_d_ind[i_cell] = v_rate_constants[offset_rc + i_rxn * L + i_cell];
+          std::fill(d_rate_d_ind.begin(), d_rate_d_ind.end(), 0);
```

Comment:
is this necessary?

Reply:
Originally the vector `d_rate_d_ind` was created in the exact location (inside double nested `for` loop). I got it out of the loops to avoid keeping hitting the heap so It has to get refreshed to zero for every iteration. Maybe it makes more sense to put it at the end of the loop. If my intention doesn't seem straightforward, I am happy to get the original version back. I'd prefer readability for this case. What do you think?

Reply:
I think it actually doesn't need this. Removed, thanks for catching it!